android/fastlane/metadata/android/en-GB/changelogs/10032.txt

```diff
@@ -0,0 +1,13 @@
+Added
+    - Basic search page for the app, accessible from the home page
+    - Users with admin privileges now have a place in the app where they can respond to requests for editing permissions
+
+Improved
+    - Web version looks a lot nicer⭐
```

Comment:
It doesn't, not yet. Plus this is a changelog for Play Store so it's not really relevant :)